fem/pbilinearform.hpp

```diff
@@ -32,7 +32,7 @@ class ParBilinearForm : public BilinearForm
    ParFiniteElementSpace *pfes; ///< Points to the same object as #fes
 
    /// Auxiliary objects used in TrueAddMult().
-   mutable ParGridFunction X, Y;
+   mutable ParGridFunction X, Y, Ytmp;
```

Comment:
`Ytmp` has to be a T-vector,  i.e. size = `GetTrueVSize()`, not a `ParGridFunction`.

Reply:
Deal!